src-api/services/db/_chats-db.ts

```diff
@@ -145,9 +146,12 @@ const renameChat = async (chat: DbChat) => {
 			...messages.map((msg: DbMessage) => ({ text: msg.text, role: msg.role })),
 			{ text: 'Title this chat concisely in its spoken language, no quotes', role: 'user' },
 		]
-		const has4oMini = await ChatGPT4oMini.isAvailable()
+		let resp
 
-		if (has4oMini) return await ChatGPT4oMini.getResponse(config, gptMessages)
+		if (!resp && (await ChatGPT4oMini.isAvailable())) resp = await ChatGPT4oMini.getResponse(config, gptMessages)
+		if (!resp && (await Claude35Haiku.isAvailable())) resp = await Claude35Haiku.getResponse(config, gptMessages)
```

Comment:
`await` inside an `if` statement. It's the first time when I am seeing it, in a codebase.
I would ask an AI, if it is correct..

Reply:
it works :)))

Reply:
But I will refactor than in next PR anyway.